src/index.html

```diff
@@ -21,7 +21,449 @@
     />
```

Comment:
The link tag at line 18-21 is linking to a .scss file. Browsers cannot interpret SCSS files, they only understand CSS. You should compile your SCSS to CSS and link the CSS file in your HTML.